### ISSUE TITLE
Some reimbursements should be substracted from the group spendings

### ISSUE THREAD

**Issue by TimonTrukker** [scastiel/spliit2#274]
When a reimbursement is done between people of the group to settle debts, nothing should happen on the group spendigs statistics.
But sometimes an external reimbursement is done for the entire group (like a deposit) which should be substracted from the group spendings statistics.

**Comment by fabfabsto** [comment]
I dont think it should be substracted from the spendings as the spendings are no longer correct. I would make this kind of payment neutral in spendings. It should only affect the balance.
